requirements/requirements.txt

```diff
@@ -1,6 +1,7 @@
 onnx
 pandas
 tensorboardX
+tensorboard
```

Comment:
what is that? the name of package is tensorboardX, not tensorboard, right?

Reply:
with torch we can use tensorboard from torch.utils. ExpManager is using torch.utils but for that tensorboard has to be installed. While in some examples tensorboardX is still in use. May be we should standardize on what to be used for tensorboard logging in the future.